docs/building.md

```diff
@@ -264,7 +318,7 @@ $ jsrun -p 2 -g 1 -c 20 -M "-gpu" ./example/advection/advection-example -i ../ex
 ```bash
 # configure and build
 $ mkdir build-cuda && cd build-cuda
-$ cmake -DCMAKE_BUILD_TYPE=Release -DMACHINE_CFG=${PARTHENON_ROOT}/cmake/machinecfg/Summit.cma
```

Comment:
I think there's an extra linebreak here (the missing `ke` is on the next line)

Reply:
oh, doh, yep.